utils.go

```diff
@@ -167,7 +167,7 @@ func (opt *flags) setOption(args []string) {
 	v.SetConfigName("jira") // config file name without extension
 	v.SetConfigType("yaml")
 
-	if configFilePtr != nil {
+	if configFilePtr != nil || len(*configFilePtr) == 0 {
```

Comment:
I'm not understanding this conditional statement. Why do you need those 2 checks? please elaborate